tests/Feature/CustomTourViewerTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\CustomTour;
+use Aic\Hub\Foundation\Testing\FeatureTestCase as BaseTestCase;
+
+class CustomTourViewerTest extends BaseTestCase
+{
+    protected $seed = true;
+
+    /**
+     * A test to check whether a randomly selected custom tour's title renders
+     * on the viewer page.
+     */
+    public function test_custom_tour_title_renders(): void
+    {
+        $randomCustomTour = CustomTour::inRandomOrder()->first();
```

Comment:
Rather than picking a random tour each time (which could leave to randomly failing/passing tests) these should explicitly select one particular tour from the seeder content and act on that.

Reply:
Thanks, I've updated this and now find the tour based on it's title.